File: application/models/DetailUtilite.php
<?php
    class DetailUtilite extends MY_Model{
        public function insert($id_utilite,$id_publication){
            $sql="INSERT INTO Detail_utilite(id_detail_utilite,id_publication,id_utilite) VALUES (default,%s,%s)";
            $sql=sprintf($sql,$this->db->escape($id_publication),$this->db->escape($id_utilite));
            $this->db->query($sql);
        }

        public function get_utilities(){
            $sql = "SELECT * FROM utilite";
            $query = $this->execute_query($sql);
            return $query->result_array();
        }
    }
?>

File: database/python/Detail_utilite.py
from random import randrange

f = open("detail_utilite.sql", "a")

id_publication = 1

while id_publication <= 500 :
    nb_utilite = randrange(1,4)
    i = 0
    while i < nb_utilite:
        id_utilite = randrange(1,4)
        f.write(f"INSERT INTO Detail_utilite(id_utilite,id_publication) values({id_utilite},{id_publication}); \n")
        i += 1
    id_publication += 1
f.close()